org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/WInfoWindowListItemRenderer.java

```diff
@@ -156,7 +157,15 @@ public String get_ValueAsString(String variableName) {
 					String zclass = styleStr.substring(MStyle.ZCLASS_PREFIX.length());
 					listcell.setZclass(zclass);
 				} else {
-					listcell.setStyle(styleStr);
+					
+					String prevStyle = listcell.getStyle();
+										
+					if(!Util.isEmpty(prevStyle)) {
+						listcell.setStyle(prevStyle + ";" + styleStr);
+					}
+					else {
+						listcell.setStyle(styleStr);
+					}
 				}
 			}
 		}

```

Comment:
Hi, just recall that we has an existing helper method for this - ZkCssHelper.appendStyle(component, style). Should use that instead.

Reply:
Hello, thank you, I will check it out.